conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -16,6 +16,10 @@ GET         /file-upload/test-only/download-file/envelopes/:envelopeId/files/:fi
 GET         /file-upload/test-only/events/:streamId                                             @uk.gov.hmrc.fileupload.testonly.TestOnlyController.getEvents(streamId: String)
 GET         /file-upload/test-only/files/inprogress                                             @uk.gov.hmrc.fileupload.testonly.TestOnlyController.filesInProgress
 
+POST        /file-upload/test-only/recreate-quarantine                                          @uk.gov.hmrc.fileupload.testonly.TestOnlyController.recreateQuarantine
+
+POST        /file-upload/test-only/recreate-collections                                         @uk.gov.hmrc.fileupload.testonly.TestOnlyController.recreateTransient
```

Comment:
lets create one endpoint only: recreate-collections which recreates quarantine AND transient